firmware/ec/src/devices/ocmp_wrappers/ocmp_powerSource.c

```diff
@@ -26,7 +26,11 @@ static bool _get_status(void *driver, unsigned int param_id, void *return_buf)
     return ret;
 }
 
-static ePostCode _probe(void *driver)
+/* Parameters are not used as this is just used to test assigning the
```

Comment:
Can you update the comment to reflect the reason that there is an unused parameter in this function? I think it's just to match the defined function type, right?